CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change Log for corePKCS11 Library
 
 ## v3.0.2 (July 2021)
+* [#118](https://github.com/FreeRTOS/corePKCS11/pull/118) Update version numbers and add C++ header guards
```

Comment:
```suggestion
* [#118](https://github.com/FreeRTOS/corePKCS11/pull/118) Add support for using library in C++ applications```

Reply:
I don't think this change is necessarily required to use the library in C++ applications